pages/blog/index.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <b-container data-qa="blog">
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <h1>{{ $t('blog.blogs') }}</h1>
+      </b-col>
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          data-qa="blog posts"
+        >
+          <ContentCard
+            v-for="(post, index) in posts"
+            :key="index"
+            :title="post.fields.headline"
+            :url="{ name: 'blog-all', params: { pathMatch: post.fields.identifier } }"
+            image-url="https://picsum.photos/600/300/?image=25"
+            :description="post.fields.description"
+            :datetime="post.fields.datePublished"
+          />
+        </b-card-group>
+      </b-col>
+    </b-row>
+    <b-row>
+      <b-col>
+        <PaginationNav
+          v-if="showPagination"
+          v-model="page"
+          :limit="perPage"
+          :total-results="total"
+          :per-page="perPage"
+          :link-gen="paginationLink"
+        />
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+
+<script lang="ts">
+  import createClient from '../../plugins/contentful';
+  import ContentCard from '../../components/generic/ContentCard';
+  import PaginationNav from '../../components/generic/PaginationNav';
+
+  export default {
+    name: 'BlogFoyer',
+
+    components: {
+      ContentCard,
+      PaginationNav
+    },
+
+    head() {
+      return {
+        title: 'Blog'
```

Comment:
Can you use the translation of Blogs here as used in the `<h1>`?

Reply:
Shall do :)